internal/project/types.ts

```diff
@@ -30,6 +30,10 @@ export type ProjectDefinition = {
 	initialized: boolean;
 };
 
+export type DependenciesExceptions = {
+	invalidLicenses: Map<string, Array<string>>;
```

Comment:
We might actually want this type to be something like:

```suggestion
	invalidLicenses: Map<string, {name: string, range: SemverRange}>;
```

That way we can support semver ranges here. Later when performing the `inConfig` check, you can pass the already parsed `version` semver AST into `satisfiesSemver` with this range.